backend/src/main/java/com/crm/tools/emails/EmailGenerator.java

```diff
@@ -0,0 +1,25 @@
+package com.crm.tools.emails;
+
+
+import com.crm.model.db.AccountEntity;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@Component
+public class EmailGenerator {
+
+    @Value("${email.endpoint.path}")
+    private String endPointPath;
+
+    public MyEmail generateEmail(AccountEntity accountEntity) {
+        return MyEmail.builder()
+                .recipient(accountEntity.getEmail())
+                .subject("Witaj " + accountEntity.getLogin() + " w aplikacji CRM")
```

Comment:
Change from polish to english everywhere, to stick to the same convention

Reply:
We are aiming polish customer